scripts/aem.js

```diff
@@ -574,7 +574,12 @@ async function fetchPlaceholders(prefix = 'default') {
  * @param {Element} main The container element
  */
 function updateSectionsStatus(main) {
-  const sections = [...main.querySelectorAll(':scope > div.section')];
+  let sections = [];
```

Comment:
There should be no need to change anything in this file.

Reply:
@bstopp Currently, with the way the sidebar logic is implemented, I need to modify this function to avoid CLS issues. The content above the fold is loading late, while blocks below the fold are loading first, causing a significant layout shift when the above-the-fold content eventually loads. Additionally, we can't directly set the height of that above-the-fold content.